synfig-studio/src/gui/widgets/widget_waypoint.cpp

```diff
@@ -254,8 +254,9 @@ void Widget_Waypoint::set_valuedesc(synfigapp::ValueDesc& value_desc)
 
 	value_widget->set_value_desc(value_desc);
 	synfig::ParamDesc param_desc;
-	if (value_desc.find_param_desc(param_desc))
-		value_widget->set_param_desc(param_desc);
+	if (value_desc.parent_is_layer())
```

Comment:
It looks like it should be:
```
	if (value_desc.parent_is_layer()) {
		value_desc.find_param_desc(param_desc);
		value_widget->set_param_desc(param_desc);
        }
```
?

Reply:
I thought so, but we use a lot of dialogs that are kept hidden/shown during the whole  application lifetime. So I thought it would be safer to reset param_desc.